src/pkg/mcp/auth/auth.go

```diff
@@ -0,0 +1,154 @@
+package auth
```

Comment:
This file is a duplicate of `./src/pkg/github/auth.go` isn't it? Can we use that one instead?